### DISCUSSION TITLE
關於髒話的研究

### DISCUSSION THREAD

**Discussion by roma0323** [1101-GS4719-C/Python-Community-GS4719-C#1562] [General]
我平常是個很容易講髒話的人，我認為講髒話雖然聽起來不太優雅，但也是有其效用的，我找了一些有關髒話的研究，覺得蠻有趣的

第一個研究，是以簡單的問卷​
調查一個人的粗口頻率及誠信程度（自評）​
結果發現，那些經常問候您家中長輩​
並關心他們性器官使用情況的人​
對自己的誠信大多更有自信​
​
研究二則是透過臉書數據進行分析​
──人們說謊時常常有一些共通跡象​
像減少第一人稱（我、恁爸、老娘等）​
或增加負面情緒詞彙（痛心、悲哀等）​
而結果顯示，髒話用量比較大的用戶​
在貼文中也比較少表現出上述特徵​
​
研究三除了使用臉書的髒話數據​
還參考一份「美國各州誠信調查」​
兩相比對後發現，那些臉書上的髒話大戶​
有大多數正是來自美國最誠信的三個州

**Comment by larry40305** [comment]
每個人對語言的選擇和表達方式都有不同的偏好和觀點，所以對於使用髒話的程度，應該考慮到場合和對方的接受度。

**Comment by aa103967622** [comment]
我覺得人很多時候在使用髒字的情況，多半是作為語助詞，一般不會有太針對性的攻擊傾向，像是單一個字的「幹」，同時可以表達很多意思，比如：幹你怎麼這樣啦，就是表示傻眼、無奈的生氣，不想原本髒字的意思可能是用性方面的字眼羞辱人。

**Comment by lyien1021** [comment]
之前也有看過說法是罵髒話可以抒發情緒，所以如果不要在公開場合，自己私底下罵其實是不錯的抒發方式。

**Comment by zhiron544** [comment]
我覺得這篇文章很有趣，也讓我對髒話有了不同的看法。我以前覺得髒話很粗俗，但現在覺得它也是一種語言表達的方式，只要不過度使用或傷害別人，也可以有正面的效果。
